source/iterators.tex

```diff
@@ -4998,7 +5009,33 @@
 ++*this;
 return tmp;
 \end{codeblock}
-Otherwise, equivalent to: \tcode{return get<I>(v_)++;}
+Otherwise, if
+\tcode{requires (I\& i) \{ \{ *i++ \} -> \exposconceptnc{can-reference}; \}}
+is \tcode{true} or
+\tcode{constructible_-\newline from<iter_value_t<I>, iter_reference_t<I>>}
```

Comment:
```suggestion
\tcode{\libconcept{constructible_-\newline from}<iter_value_t<I>, iter_reference_t<I>>}
```
Does this work?

Reply:
I don't think so. Let's skip this.

Reply:
I'd like to see this fixed also. I assume the \newline was to address an Overfull? Did you try "constructible_\-from" or "constructi\-ble_from"? If it doesn't work inside the tcode (don't know why it wouldn't), something like the following should work:
`\libconcept{constructible_\-from}\tcode{<iter_value_t<I>, iter_reference_t<I>>}`